pkg/util/cmdutil/console.go

```diff
@@ -93,3 +94,110 @@ func RemoveTralingNewline(s string) string {
 
 	return s
 }
+
+// TableRow is a row in a table we want to print.  It can be a series of a columns, followed
+// by an additional line of information.
+type TableRow struct {
+	Columns        []string
+	AdditionalInfo string
+}
+
+// PrintTable prints a grid of rows and columns.  Width of columns is automatically
+// determined by the max length of the items in each column.
+func PrintTable(table []TableRow) {
+	if len(table) == 0 {
```

Comment:
Nit, this didn't seem necessary -- if it was 0 width, we'd just do the right thing based on the logic below.